### PR TITLE
feat: update to Facebook iOS SDK 9.0.0

### PR DESCRIPTION
**Pull request by hansemannn**
Started with an update to the SDK 7.0.0, then 8.0.0 and now (01/20/2021) to 9.0.0.

!! Includes the new `loginTracking` property for improved privacy !!

[facebook-iphone-10.0.0.zip](https://www.dropbox.com/s/fo49gi291kv6jo3/facebook-iphone-10.0.0.zip?dl=1)

In addition to Android via #156

JIRA: https://jira.appcelerator.org/browse/MOD-2872

### CODE REVIEW

File: ios/titanium-facebook.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -343,7 +343,7 @@
 				GCC_WARN_UNUSED_VALUE = NO;
 				GCC_WARN_UNUSED_VARIABLE = NO;
 				INSTALL_PATH = /usr/local/lib;
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
```
Comment (vijaysingh-axway) [inline ios/titanium-facebook.xcodeproj/project.pbxproj]: In titanium SDK deployment target is 10.0. Same should be here.

Comment (hansemannn) [reply ios/titanium-facebook.xcodeproj/project.pbxproj]: Oh, I thought it's being bumped to 11 in SDK 10. But sure, will do!

**Review by vijaysingh-axway — APPROVED** [review]
CR passed.

**Comment by ssekhri** [comment]
FR Passed

**Pull request merged**